### ISSUE TITLE
Geolocalisation IP

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kalnyc67/analytics-issues#196]
```
---------------------------------------------------------------------------
NOTE: This issue tracking system is for Google Analytics developer products
only.
If you are not a developer/programmer visit:
http://www.google.com/analytics/support.html
---------------------------------------------------------------------------
Name of related component: Tracking Code (Javascript / ga.js)

Request summary:
Provide a brief summary of the the feature you'd like to see implemented.

Use Cases:
I would like to track file downloads from my site, but unfortunately, not all 
the downloads are done via a webbrowser.  Typically, a certain number of file 
are download using external program (typically "wget" command or "java" 
software).  I am also automatically generating content for download using php 
scripts.  For these reasons, it is really hard for me to use javascripts in all 
the case.  (of course, I do use the standard ga.js script on normal web page, 
but can't do it for direct download).
For the later, I have basically no other choice than using a php based tracking 
(http://code.google.com/p/php-ga/).  I discovered the php-ga project that is 
basically an interface between ga server side and php scripts.  All works very 
well at the exception of the geolocalisation.

The problem is that the GA-geolocalisation will rely on the IP-address of the 
GA client (which is in this case the IP-address of the PHP server instead of 
the one from the web "visitor").  Would it be possible to change this and allow 
to use the IP address of the visitor.  For instance using a field to define the 
ip address to be used for geoloc.

Thanks in advance,
Loic



```

Original issue reported on code.google.com by `loic.que...@gmail.com` on 7 Mar 2012 at 10:34
